src/lib/db-listeners/db-comments-listener.ts

```diff
@@ -0,0 +1,53 @@
+import { DB } from "../db";
+import { TileCommentModel, TileCommentsModelType, TileCommentsModel } from "../../models/tools/tile-comments";
+import { forEach } from "lodash";
+
+export class DBCommentsListener {
+  private db: DB;
+  private commentsRef: firebase.database.Reference | null = null;
+
+  constructor(db: DB) {
+    this.db = db;
+  }
+
+  public start() {
+    this.commentsRef = this.db.firebase.ref(
+      this.db.firebase.getUserDocumentCommentsPath(this.db.stores.user)
+    );
+    this.commentsRef.on("child_changed", this.handleUpdateComments);
+    this.commentsRef.on("child_added", this.handleUpdateComments);
```

Comment:
In a prior conversation we were musing that `child_added` might never be sent, given that the listener is higher up, adding comments might result in `child_changed` events instead. I would think that if `child_added` is necessary, `child_removed` would also be necessary.